### ISSUE TITLE
Ionic Foreign Keys

### ISSUE THREAD

**Issue by jondediego** [typeorm/typeorm#1053]
Hi,
I am creating a ManyToOne relation :
```
@Entity()
export class User{
    @PrimaryGeneratedColumn()
    Id: number;

    @ManyToOne(type => Workplace)
    Workplace: Workplace;
}

@Entity()
export class Workplace extends BaseEntity {
   @PrimaryGeneratedColumn()
    Id: number;
}
```

Then I insert workplaces in database. After that I try to insert a user with a workplace.Id that doesn't exist for workplace table. The insertion is done althought the foreign key einserted is not correct:
` let user: User = new User();
const workplace = new Worplace();
workplace.Id = 143214321432
        user.Workplace = workplace;
userRepository.save(user);`

I think that in sqlite databases, foreign keys are disabled by default in order to achieve retrocompatibility. So, in your sqlite drivers you run PRAGMA foregn_keys=ON, in order to enable foreign keys. This isn't done in your cordova driver. If I execute that sentence through ORM connection, foreign keys are enabled and user with wrong workplace isn't inserted in database:
` await this.connection.manager.query("PRAGMA foreign_keys=ON;");`

Could you add this to your cordova drivers? Would it work for WebSQL also?

**Comment by daniel-lang** [comment]
Hey, I must have forgotten to add that in the cordova driver. I will add it.
As far as I know WebSQL doesn't support PRAGMA statements, therefore it wont work there.

**Comment by daniel-lang** [comment]
The fix will be released with 0.1.2
